pkg/config/default.go

```diff
@@ -242,7 +242,9 @@ const (
 	// DefaultServiceRouteReporter .
 	DefaultServiceRouteReporter string = "serviceRoute"
 	// DefaultStatReportEnabled .
-	DefaultStatReportEnabled bool = false
+	DefaultStatReportEnabled bool = true
```

Comment:
for `DefaultStatReportEnabled bool = true` const is defined here without writing the type, go will automatically infer the type. It is recommended to write directly in the following style.
```go
const (
	// DefaultStatReporter .
	DefaultStatReporter = "stat2Monitor"
	// DefaultCacheReporter .
	DefaultCacheReporter = "serviceCache"
	// DefaultPluginReporter .
	DefaultPluginReporter = "pluginInfo"
	// DefaultLoadBalanceReporter .
	DefaultLoadBalanceReporter = "lbInfo"
	// DefaultRateLimitReporter .
	DefaultRateLimitReporter = "rateLimitRecord"
	// DefaultServiceRouteReporter .
	DefaultServiceRouteReporter = "serviceRoute"
	// DefaultStatReportEnabled .
	DefaultStatReportEnabled = true
	// DefaultMetricsChain .
	DefaultMetricsChain = "prometheus"
)
```